NeosModLoader/ModLoader.cs

```diff
@@ -236,6 +237,7 @@ private static LoadedNeosMod InitializeMod(ModAssembly mod)
                     Logger.ErrorInternal($"unexpected null instantiating mod {modClass.FullName} from {mod.File}");
                     return null;
                 }
+                SplashChanger.SetCustom($"Loading mod [{neosMod.Name}/{neosMod.Version}]");
```

Comment:
At this point we're already mostly done loading the mod, and all that remains is the mod configuration. I'd change this from `Loading mod` to `Loading mod configuration`. I'd also add another SplashChanger.SetCustom() up in the LoadAssembly() method right before the `assembly = Assembly.LoadFile(mod.File);` line, as that part can take a long time (especially if your antivirus decides to angrily glare at all the sussy DLL files).